tests/integration/hive_metastore/test_catalog_schema.py

```diff
@@ -16,6 +17,36 @@
 _SPARK_CONF = get_azure_spark_conf()
 
 
+@pytest.fixture
+def catalog_name(ws, make_random) -> Iterator[str]:
+    name = f"test-catalog-{make_random(5)}"
+    yield name
+    try:
+        ws.catalogs.delete(name, force=True)
+        logger.info(f"Deleted test catalog: {name}")
+    except NotFound:
+        pass
+
+
+@retried(on=[NotFound], timeout=timedelta(minutes=2))
+def test_create_ucx_catalog_creates_catalog(
+    ws,
+    runtime_ctx,
+    watchdog_remove_after,
+    catalog_name,
+) -> None:
+    prompts = MockPrompts({f"Please provide storage location url for catalog: {catalog_name}": "metastore"})
+    runtime_ctx.catalog_schema.UCX_CATALOG = catalog_name
+
+    runtime_ctx.catalog_schema.create_ucx_catalog(prompts, properties={"RemoveAfter": watchdog_remove_after})
+
```

Comment:
I'm glad to see we're [updating pytester](https://github.com/databrickslabs/pytester/pull/52) to handle this.

Reply:
To be precise: we are not updating pytester to handle this. The update allows us to pre-create a catalog for the tests. In this test we want to start with **no** catalog so that we can test creating one.

Reply:
Ah okay: thanks for the clarification.